### ISSUE TITLE
Missing License Disallows Free Use

### ISSUE THREAD

**Issue by l-zeuch** [SwarajEPICoder/CO2Emissions#1]
Hey there!
Stumbled upon this repository and wanted to use it as a source of one of my research projects, however due to a missing license agreement, this is sadly not possible.

After a quick search I've found [CC-BY-SA-4.0](https://choosealicense.com/licenses/cc-by-sa-4.0/) to be the most fitting license - this is also what the vast majority of encyclopediae, such as wikipedia, use.

Alternatively, you can use [this tool](https://choosealicense.com "License Helper Tool") to pick the most fitting - however please note that a software license (most common software licenses are `MIT`, any GPL, `BSD-3-Clause` et al.) cannot be applied to raw data/text, nor code snippets. 
As with the jupyter notebooks: This is quite finnicky and rather complicated, due to its dependency on your intent. As far as I can tell, your main intent is to share data rather than code, hence a software license is unlikely to be applicable - should this not be the case, and you instead planned to share code, you might as well apply a software license.

This [stackexchange](https://opensource.stackexchange.com/questions/5052/do-i-need-to-license-a-jupyter-notebook-that-uses-a-forked-agpl-library#5366) post on opensource.stackexchange could give you more insight to this, although it might as well overcomplicate a few things - read at your own risk!
The TL;DR of this is in general: Declare two licenses - one for the data, one for the code.

Thanks in advance!
